tests/js/spec/views/dashboardsV2/widgetBuilder/widgetBuilder.spec.tsx

```diff
@@ -20,6 +21,8 @@ import {
 import * as dashboardsTypes from 'sentry/views/dashboardsV2/types';
 import WidgetBuilder, {WidgetBuilderProps} from 'sentry/views/dashboardsV2/widgetBuilder';
 
+jest.mock('sentry/actionCreators/modal');
```

Comment:
@priscilawebdev I moved `openWidgetBuilderOverwriteModal` back to `sentry/actionCreators/modal` because it made it easier to mock. Not sure if you know of a way to mock the function inside the module but I tried:
```js
const mockOpenModal = jest.fn();
jest.mock('sentry/views/dashboardsV2/widgetBuilder/widgetLibrary', () => ({
  ...jest.requireActual('sentry/views/dashboardsV2/widgetBuilder/widgetLibrary'),
  openWidgetBuilderOverwriteModal: mockOpenModal,
  __esModule: true
}))
```

and different variations without luck (i.e. removing props from that return object, appending `/index` to the mock path).

Reply:
I also tried a couple of other things but didn't work... now I'm curious and will take a look into it afterward... but let's merge as it is. Thanks, Nar :)